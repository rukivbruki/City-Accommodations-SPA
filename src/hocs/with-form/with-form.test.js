import React from 'react';
import renderer from 'react-test-renderer';
import withForm from './with-form';

const MockComponent = () => <div />;
const MockComponentWrapped = withForm()(MockComponent);

it(`withForm snapshot`, () => {
  const tree = renderer.create(<MockComponentWrapped />).toJSON();
  expect(tree).toMatchSnapshot();
});
