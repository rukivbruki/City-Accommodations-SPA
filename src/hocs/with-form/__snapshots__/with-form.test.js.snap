// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withForm snapshot 1`] = `<div />`;
